frontend/public/components/service-account.jsx

```diff
@@ -110,11 +110,11 @@ const menuActions = [
 const kind = 'ServiceAccount';
 
 const tableColumnClasses = [
-  classNames('col-sm-4', 'col-xs-6'),
-  classNames('col-sm-4', 'col-xs-6'),
-  classNames('col-md-2', 'hidden-sm', 'hidden-xs'),
-  classNames('col-sm-2', 'hidden-xs'),
-  Kebab.columnClass,
+  'pf-u-w-50-on-sm pf-u-w-33-on-md pf-u-w-25-on-lg',
+  'pf-u-w-25-on-md',
+  'pf-m-hidden pf-m-visible-on-lg pf-u-w-25-on-lg',
+  'pf-m-hidden pf-m-visible-on-md pf-u-w-25-on-md',
+  `${Kebab.columnClass} pull-right`,
```

Comment:
@abhinandan13jan I think pull-right is an indicator that we waste space for the kebab/action icon column. I like that the most other changes has at least one column without a width, which takes over "all leftover" space.

Service accounts on a big screen before:
![image](https://user-images.githubusercontent.com/139310/123612997-59eae580-d803-11eb-9cdd-9298acfa3e38.png)

On a small screen before:
![image](https://user-images.githubusercontent.com/139310/123613535-ccf45c00-d803-11eb-8b5b-76f5c0e4aed9.png)

With this PR on a big screen:
![image](https://user-images.githubusercontent.com/139310/123613408-afbf8d80-d803-11eb-8bb3-dcb3a086eb3f.png)

With this PR on a small screen (you can see the 25% space for the action column):
![image](https://user-images.githubusercontent.com/139310/123613641-e5647680-d803-11eb-8b52-5bc9f1e3ee83.png)

I would suggest a css without styling for the name and without `pull-right`.

```suggestion
  '',
  '',
  'pf-m-hidden pf-m-visible-on-lg pf-u-w-25-on-lg',
  'pf-m-hidden pf-m-visible-on-md pf-u-w-25-on-md',
  Kebab.columnClass,
```

It looks like this then (yes, also not similar to the original design) on a big screen:
![image](https://user-images.githubusercontent.com/139310/123613239-8d2d7480-d803-11eb-9eb8-f24b788b37c8.png)

And like this on a small screen:
![image](https://user-images.githubusercontent.com/139310/123613872-1ba1f600-d804-11eb-8b72-b71a9e0ffe7f.png)

Wdyt?

Reply:
updated